### DISCUSSION TITLE
Can't click Save in setting window.

### DISCUSSION THREAD

**Discussion by abab6111** [apohl79/audiogridder#1250] [Help]
![image](https://github.com/apohl79/audiogridder/assets/11353860/2b64141c-bb25-4525-8bf7-e50ecbc3c762)
I can't slide down to click Save.
I need to change the size of text to 100% to click Save.
Can the window automatically adjust its size or allow scrolling downwards?

**Comment by FAWN-MUSIC** [comment]
Happens to me too.
Ctrl+alt+ Left arrow rotate the display on it's side so that you can see the bottom of the settings panel and the save button.
Ctrl+alt +Down arrow to bring your display orientation back to normal
